src/gui/ogr/qgsvectorlayersaveasdialog.cpp

```diff
@@ -39,7 +39,7 @@ static const int COLUMN_IDX_EXPORT_AS_DISPLAYED_VALUE = 2;
 
 QgsVectorLayerSaveAsDialog::QgsVectorLayerSaveAsDialog( long srsid, QWidget *parent, Qt::WindowFlags fl )
```

Comment:
Can you deprecate this constructor? I want to completely eliminate srs ids in qgis 4.0...

Reply:
done